keyboards/massdrop/alt/keymaps/elijahblake/rules.mk

```diff
@@ -0,0 +1,16 @@
+# RGBLIGHT_ENABLE = no				# Not for MD boards. This is here in case you forget.
+COMMAND_ENABLE = yes				# Commands for debug and configuration
+# BOOTMAGIC_ENABLE = full			# Virtual DIP switch configuration
+# BOOTMAGIC_ENABLE = lite
+# AUTO_SHIFT_ENABLE = yes      	# Auto Shift
+NKRO_ENABLE = yes           	# USB Nkey Rollover
+DYNAMIC_MACRO_ENABLE = yes		# Dynamic macro recording and play
+#MOUSEKEY_ENABLE = yes			# Enable mouse control keycodes. Increases firmware size.
+#TAP_DANCE_ENABLE = yes			# Enable tap dance keys
+#CONSOLE_ENABLE = yes			# Enable debugging console. Increases firmware size.
+SRC += config_led.c				# Used to add files to the compilation/linking list.

```

Comment:
```suggestion
```

Reply:
Taking these lines out broke the build for me.

Reply:
did clean make and it was successful.  Applied these changes.